### PR TITLE
Csrf protection rails backbone

### PR DESCRIPTION
**Pull request by schmierkov**
@etagwerker @mauro-oto please check this out, I hope I generated everything correctly.

### CODE REVIEW

File: _posts/2015-12-16-adding-csrf-protection-to-your-rails-backbone-app.markdown [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+layout: post
+title: "Adding Csrf-Protection to your Rails-Backbone App"
+date: 2015-12-16 03:55:00
+categories: ["rails", "backbone", "security"]
+author: "schmierkov"
+---
+
+When integrating Backbone.js in your Rails App, you might face the problem of the inability to verify the CSRF-Token.
+
+The CSRF Protection secures your app with a token to make sure, that the person who is interacting with it is you. So you should not turn it off, except you know what you are doing. For more informations on this Topic, check out the [Rails Security Guide](http://guides.rubyonrails.org/security.html#cross-site-request-forgery-csrf).
```
Comment (mauro-oto) [inline _posts/2015-12-16-adding-csrf-protection-to-your-rails-backbone-app.markdown]: @schmierkov I'd change `except you know what you are doing` for `unless you know what you are doing`. And `informations` should be just `information`. :+1:

File: _site/LICENSE.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2013 Ghost Foundation - Released under The MIT License.
+Copyright (c) 2015 The Lean Software Boutique LLC - Released under The MIT License.
```
Comment (mauro-oto) [inline _site/LICENSE.txt]: @schmierkov nice :+1:

File: _posts/2015-12-16-adding-csrf-protection-to-your-rails-backbone-app.markdown [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      var token = $('meta[name="csrf-token"]').attr('content');
+      if (token) { xhr.setRequestHeader('X-CSRF-Token', token); }
+      if (beforeSend) { return beforeSend.apply(this, arguments); }
+    };
+  }
+  return Backbone._sync(method, model, options);
+};
+```
+
+It grabs the csrf-token provided in the meta tags of your rails application and sets this as request header field `X-CSRF-Token`.
```
Comment (mauro-oto) [inline _posts/2015-12-16-adding-csrf-protection-to-your-rails-backbone-app.markdown]: @schmierkov I'd stick either to `CSRF-token` or `csrf-token`. Also capitalize Rails where it's not capitalized, there's currently a mix of both of these.

**Comment by mauro-oto** [comment]
@schmierkov the article looks good, and the generated site looks correct! Check the comments I left and re-generate the site afterwards. :+1:

**Comment by schmierkov** [comment]
@mauro-oto thanks for your feedback! :+1:

**Comment by mauro-oto** [comment]
@schmierkov awesome :beers: 

Thanks!

**Pull request merged**